sqlparser/sqlparser.go

```diff
@@ -223,6 +224,27 @@ func (p *Parser) replaceInsertValueFromValArg(query *InsertQuery, colIndex int,
 				Val:  []byte(fmt.Sprintf("%d", arg)),
 			}
 		}
+	case time.Time:
```

Comment:
May collect cases of `time.Time` and `*time.Time` ?

Reply:
https://github.com/knocknote/octillery/pull/17/commits/f87c53632ad01c8921970311c37cfc11c996734d